apps/registry/next.config.js

```diff
@@ -28,4 +28,16 @@ module.exports = {
   compiler: {
     styledComponents: true,
   },
+  webpack: (config) => {
+    config.module.rules.push({
+      test: /\.hbs$/,
+      loader: 'handlebars-loader',
+      options: {
+        precompileOptions: {
+          knownHelpersOnly: false,
+        },
+      },
+    });
+    return config;
+  },
```

Comment:
So this part is kinda bad, I find. Unfortunately I did not find a way to specify the additional loader in a webpack config for the `papirus` theme and now the information that this package is using handlebars is leaking out. I find it still better than to have to wrap all `.hbs` and potentially `.css` files content in javascript and lose all syntax hightlighting, etc.

Reply:
FYI: Two of the templates were buggy! A closing div was missing in [`resume.hbs`](https://github.com/konalexiou/jsonresume-theme-papirus/blob/355beb2daff6372fdb1f64d64e3f4e91098d808b/resume.hbs#L35) and [this](https://github.com/konalexiou/jsonresume-theme-papirus/blob/355beb2daff6372fdb1f64d64e3f4e91098d808b/partials/interests.hbs#L21) `<ul>` in `interests.hbs` should be closing tag like `</ul>`. I fixed this, which was easy since `prettier` told me to.